zcash_client_backend/Cargo.toml

```diff
@@ -20,12 +20,17 @@ base64 = "0.13"
 ff = "0.11"
 group = "0.11"
 hex = "0.4"
+hdwallet = { version = "0.3.0", optional = true }
 jubjub = "0.8"
+log = "0.4"
 nom = "7"
 percent-encoding = "2.1.0"
 proptest = { version = "1.0.0", optional = true }
 protobuf = "2.20"
 rand_core = "0.6"
+ripemd160 = { version = "0.9.1", optional = true }
```

Comment:
This crate appears to be deprecated; its [crates.io entry](https://crates.io/crates/ripemd160/versions) says to use the [ripemd crate](https://crates.io/crates/ripemd/versions) instead.